packages/api/src/routers/api/team.ts

```diff
@@ -158,4 +158,48 @@ router.get('/tags', async (req, res, next) => {
   }
 });
 
+router.delete('/users/:userEmail', async (req, res, next) => {
+  try {
+    const teamId = req.user?.team;
+    const userEmail = req.params.userEmail;
+    if (teamId == null) {
+      throw new Error(`User ${req.user?._id} not associated with a team`);
+    }
+    if (userEmail == null) {
```

Comment:
we're trying to move all parameter validation via `zod` to keep our validation logic cleaner and get some nice TS typing on top ([example](https://github.com/hyperdxio/hyperdx/blob/a2c257f5eb0288af2b97be27269d5642763e9a12/packages/api/src/routers/api/logViews.ts#L12))

Reply:
Thanks for the review, changes made